### DISCUSSION TITLE
CLA for plugins

### DISCUSSION THREAD

**Discussion by W0lfgunbl00d** [audacity/audacity#7710] [Q&A] [answered]
Hello,

I have a question regarding the CLA. If I develop a plugin, can I publish it without signing the CLA, or is the CLA only required for submitting pull requests?

Thank you in advance for your clarification.

**Comment by LWinterberg** [comment, marked as answer]
The CLA is only necessary to get stuff included in the audacity repo. If you plan to distribute a plugin yourself, you don't need a CLA

**Reply by W0lfgunbl00d** [reply]
Thanks
